packages/platforms/react-native/lib/auto-instrumentation/network-request-plugin.ts

```diff
@@ -17,12 +17,16 @@ import { type ReactNativeConfiguration } from '../config'
 
 const permittedPrefixes = ['http://', 'https://', '/', './', '../']
 
+// The NetInfo module makes requests to this URL to detect if the device is connected to the internet - we don't want to track these
+// see https://github.com/react-native-netinfo/react-native-netinfo/blob/1cd754de6c1fb102a491af418e3b6e831f58855a/src/internal/defaultConfiguration.ts#L4
+const NET_INFO_REACHABILITY_URL = 'https://clients3.google.com/generate_204'
+
 export interface ReactNativeNetworkRequestInfo extends NetworkRequestInfo {
   readonly type: 'xmlhttprequest'
 }
 
 export class NetworkRequestPlugin implements Plugin<ReactNativeConfiguration> {
-  private configEndpoint: string = ''
+  private ingoredUrls: string[] = [NET_INFO_REACHABILITY_URL]
```

Comment:
`ingored` &rarr; `ignored`

```suggestion
  private ignoredUrls: string[] = [NET_INFO_REACHABILITY_URL]
```